### PR TITLE
Add build options

### PR DESCRIPTION
**Pull request by mw-kapilg**
### Changes
- Added support for `build-options`
- Minor changes to UI

### Sample screenshot
Screenshot 2024-02-21 201046

### CODE REVIEW

File: src/main/resources/matlab-bamboo-plugin.properties [hunk 1]
```diff
@@ -5,14 +5,14 @@ matlab.plugin.command=Command
...
 
 matlab.helpPath = Full path to MATLAB root folder
-matlab.options = Specify startup options
-matlab.options.field = Options
+matlab.options = Startup options
 
 # MATLAB Build keys
-matlab.build.tasks = Build tasks
+matlab.build.tasks = Tasks
+matlab.build.options = Build options
```
Comment (mw-hrastega) [inline src/main/resources/matlab-bamboo-plugin.properties:12]: Will the "Specify build options" check box appear below the "Tasks" box?

File: src/main/resources/matlab-bamboo-plugin.properties [hunk 2]
```diff
@@ -5,14 +5,14 @@ matlab.plugin.command=Command
 plugin.command.error=Command field is empty.
 
 matlab.helpPath = Full path to MATLAB root folder
-matlab.options = Specify startup options
-matlab.options.field = Options
+matlab.options = Startup options
```
Comment (mw-hrastega) [inline src/main/resources/matlab-bamboo-plugin.properties:8]: I am not sure if we can remove "Specify" from the UI. In the initial plugin design with Preethi and Anjana, we decided that "Specify" is needed for clarity and for consistency with other verb phrases, such as **Disable** this task, in the task configuration panel.

File: src/main/resources/matlab-bamboo-plugin.properties (hunk 1 above)
Comment (mw-hrastega) [inline src/main/resources/matlab-bamboo-plugin.properties:12]: Specify build options?

File: src/main/resources/matlab-bamboo-plugin.properties [hunk 3]
```diff
@@ -5,14 +5,14 @@ matlab.plugin.command=Command
...
 
 # MATLAB Build keys
-matlab.build.tasks = Build tasks
+matlab.build.tasks = Tasks
+matlab.build.options = Build options
 
 #Source folder
-matlab.test.srcfolder.exists= Specify source folder
+matlab.test.srcfolder.exists= Source folder
```
Comment (mw-hrastega) [inline src/main/resources/matlab-bamboo-plugin.properties:15]: Specify source folder

**Review by mw-hrastega — CHANGES_REQUESTED** [review]
Adding some comments and questions. It will be nice to see some screenshots including the proposed UI changes.

File: src/main/resources/matlab-bamboo-plugin.properties (hunk 2 above)
Comment (mw-kapilg) [reply src/main/resources/matlab-bamboo-plugin.properties:8]: Ohh I see, sure I'll revert these changes

File: src/main/resources/matlab-bamboo-plugin.properties (hunk 1 above)
Comment (mw-kapilg) [reply src/main/resources/matlab-bamboo-plugin.properties:12]: Yes, adding screenshot for clarity

**Review by mw-hrastega — APPROVED** [review]
Looking good. Thank you, Kapil.

**Review by davidbuzinski — APPROVED** [review]

**Pull request merged**